### ISSUE TITLE
recording app: rethink team members UI

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2920]
The team members UI in FM is a little awkward for large teams and I wonder if it could be improved. In particular it is clunky if one wants to find out if a user has already been created - one has to scroll through all the users. In doing that in BUAP just now I caught another problem: sometimes people just double-create users. That can't be triggering huge problems or we would have run into them in BUAP already where "admin" has been made three separate times (twice as "Administrator" and once as "administrator", all with login admin).

At least for the work I'm doing now, I'd prefer a grid where I can see the whole team at a glance and add people while looking at everyone else. Would also help me get a quick overview of who is a registrar, which would be helpful.

**Comment by lbestock** [comment]
But: not high priority at all, and perhaps more a thing to think about for the UI for team members when we scrap FM than something to actually intervene on now.

**Comment by urapadmin** [comment]
One of our oldest FM layouts. it is something I can do when needing a break from the complicated parts.

**Comment by lbestock** [comment]
For thought: it has to result in users in FM, but does it have to happen in FM? This is never done in the trench, always when one is sitting down with a laptop.

**Comment by urapadmin** [comment]
no, that would be way more work. As long as FileMaker is our input app we have to limit the inputs in Kiosk proper to the absolute minimum.

**Comment by urapadmin** [comment]
### recording 16.8
- new simple team member layout as a list

**Comment by lbestock** [comment]
Just saying: this is really appreciated. It helps so much.
